packages/suite/src/components/wallet/AccountsMenu/AccountItem.tsx

```diff
@@ -147,6 +151,9 @@ export const AccountItem = forwardRef((props: AccountItemProps, ref: React.Ref<H
         <DefaultLabel />
     );
 
+    // Show skeleton instead of zero balance during CoinJoin initial discovery
+    const isBalanceShown = account.backendType !== 'coinjoin' || account.status !== 'initial';
```

Comment:
Why not use this for non-coinjoin accounts as well? Discovery can take longer sometimes.

Reply:
I think for other accounts, the whle account is not rendered untill fully discovered, so it's slightly different logic. Not sure is it makes sence to push it to this PR

Reply:
I'd rather not do it if there hasn't been a demand for it by now.